lib/TraackrApi/TraackrApiObject.php

```diff
@@ -308,6 +308,6 @@ public function delete($url, $params = [])
         $logger = TraackrAPI::getLogger();
         $logger->debug('Calling (DELETE): ' . $url);
 
```

Comment:
I had to set "decode" to false or else this code incorrectly returned "false":

```
        // API MUST return UTF8
        if ($decode) {
            $rez = json_decode($curl_exec, true);
        } else {
            $rez = $curl_exec;
        }

        return null === $rez ? false : $rez;
```

Calling json_decode on an empty string returns null, which then returned "false" to the caller. Figured it's best to just return Core's empty string rather than (accidentally) converting it to "false".